index.html

```diff
@@ -0,0 +1,168 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Architects+Daughter" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+    <title>Fediuchko Elvira — Résumé</title>
+  </head>
+    <body>
```

Comment:
Use 2 spaces for indentation (https://www.w3schools.com/html/html5_syntax.asp "Blank Lines and Indentation")
//comment from